assets/prototype/application/ui/components/layout/editor-modal/useEditorModalState.ts

```diff
@@ -79,12 +85,12 @@ const useEditorModalState = () => {
 	 * @param {string} editorId    unique identifier for editor
 	 * @return {string} editorId
 	 */
-	const isEditorOpen = (editorId: string): boolean => editorId === head(state);
+	const isEditorOpen = (editorId: string): boolean => editorId === last(state.stack);
 
 	return {
 		closeEditor,
 		closeAllEditors,
-		currentlyOpenEditor,
+		getCurrentlyOpenEditor,
```

Comment:
Great analogy, thx.

Reply:
I think I messed up things along the way.
I was thinking that we want to separate modal state by ticket id, so was thinking to pass ticket id to `getCurrentlyOpenEditor` and later found out, that it isn't the case since at the same time we will have multiple modals only from the same ticket.
Will remove the `get` prefix.